### PR TITLE
modify maxmemocharlength

### PR DESCRIPTION
**Pull request by chandiniv1**
closes: #5428

### CODE REVIEW

File: modules/apps/27-interchain-accounts/types/packet_test.go [hunk 1]
```diff
@@ -7,8 +7,6 @@ import (
 	ibctesting "github.com/cosmos/ibc-go/v8/testing"
 )
 
-var largeMemo = "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum"
```
Comment (damiannolan) [inline modules/apps/27-interchain-accounts/types/packet_test.go:10]: RIP lorem ipsum!

**Review by damiannolan — APPROVED** [review]
🚀

File: modules/apps/27-interchain-accounts/types/packet_test.go (hunk 1 above)
Comment (chatton) [reply modules/apps/27-interchain-accounts/types/packet_test.go:10]: lorem ripsum

**Review by chatton — APPROVED** [review]
LGTM!

**Pull request merged**